internal/pkg/artifact/config.go

```diff
@@ -64,6 +65,42 @@ func NewReloader(cfg *Config, log *logger.Logger) *Reloader {
 }
 
 func (r *Reloader) Reload(rawConfig *config.Config) error {
+	if err := r.reloadConfig(rawConfig); err != nil {
+		return errors.New(err, "failed to reload source URI")
```

Comment:
"failed to reload source URI" ?
did you mean something like "failed to reload config"? cause the next error bellow has identical message, would not be able to tell one from another